src/patternfly/components/Menu/menu.scss

```
@@ -10,6 +10,8 @@
   --#{$menu}--BackgroundColor: var(--pf-t--global--background--color--floating--default);
   --#{$menu}--BoxShadow: var(--pf-t--global--box-shadow--md);
   --#{$menu}--Color: var(--pf-t--global--text--color--regular);
+  --#{$menu}--BorderWidth: var(--pf-t--global--high-contrast--border--width--control--default);
```

should this use the box token?

```suggestion
  --#{$menu}--BorderWidth: var(--pf-t--global--border--width--box--default);
```